piet-direct2d/src/lib.rs

```diff
@@ -532,6 +544,15 @@ fn draw_image<'a>(
     dst_rect: Rect,
     interp: InterpolationMode,
 ) {
+    let src_size = image.get_size();
+    if dst_rect.width() == 0.
```

Comment:
I think `dst_rect.is_empty()` might be clearer here.

Reply:
Sorry I missed this, will fix before merge, and I'll have a quick look to make sure I didn't miss any more.